src/main/java/io/quarkus/bot/PingWhenNeedsTriageRemoved.java

```diff
@@ -0,0 +1,125 @@
+package io.quarkus.bot;
+
+import io.quarkiverse.githubapp.ConfigFile;
+import io.quarkiverse.githubapp.event.Issue;
+import io.quarkus.bot.config.QuarkusBotConfig;
+import io.quarkus.bot.config.QuarkusBotConfigFile;
+import io.quarkus.bot.el.SimpleELContext;
+import io.quarkus.bot.util.Labels;
+import io.quarkus.bot.util.Patterns;
+import io.quarkus.bot.util.Strings;
+import org.jboss.logging.Logger;
+import org.kohsuke.github.GHEventPayload;
+import org.kohsuke.github.GHIssue;
+import org.kohsuke.github.GHLabel;
+
+import javax.el.ELContext;
+import javax.el.ELManager;
+import javax.el.ExpressionFactory;
+import javax.el.ValueExpression;
+import javax.inject.Inject;
+import java.io.IOException;
+import java.util.Set;
+import java.util.TreeSet;
+
+public class PingWhenNeedsTriageRemoved {
+    private static final Logger LOG = Logger.getLogger(PingWhenNeedsTriageRemoved.class);
+
+    @Inject
+    QuarkusBotConfig quarkusBotConfig;
+
+    void pingWhenNeedsTriageRemoved(@Issue.Unlabeled GHEventPayload.Issue issuePayload,
+            @ConfigFile("quarkus-bot.yml") QuarkusBotConfigFile quarkusBotConfigFile) throws IOException {
+
+        if (quarkusBotConfigFile == null) {
+            LOG.error("Unable to find triage configuration.");
+            return;
+        }
+
+        GHIssue issue = issuePayload.getIssue();
+        GHLabel removedLabel = issuePayload.getLabel();
```

Comment:
One of the things I learned first when I joined Red Hat is: bail out early if you can. It simplifies your logic a lot.

So in this case, you can test if the removed label is not `TRIAGE_NEEDS_TRIAGE` just here and return right away. That way you don't do anything useless and you don't have to nest conditions.